src/txmempool.h

```diff
@@ -735,10 +735,10 @@ class CTxMemPool
     void GetTransactionAncestry(const uint256& txid, size_t& ancestors, size_t& descendants, size_t* ancestorsize = nullptr, CAmount* ancestorfees = nullptr) const;
 
     /** @returns true if the mempool is fully loaded */
-    bool IsLoaded() const;
+    bool GetLoadTried() const;
```

Comment:
comment could be updated to reflect it indicates load attempt not succes

Reply:
re: https://github.com/bitcoin/bitcoin/pull/25487#discussion_r917203102

In commit "scripted-diff: Rename m_is_loaded -> m_load_tried" (ae6bf8ed30d5b8511f22d3a8f16112b33804944f)

> comment could be updated to reflect it indicates load attempt not succes

In case you do manually update the comment after the scripted diff, could also renamed the "bool loaded" argument below "bool load_tried"

Reply:
Fixed in 82111033f3